lstmcpipe/stages/mc_train.py

```diff
@@ -191,7 +191,7 @@ def train_pipe(
     #     print(cmd)
 
     # 'sbatch --parsable --dependency=afterok:{wait_ids_proton_and_gammas} -e {jobe} -o {jobo} --wrap="{base_cmd}"'
-    batch_cmd = "sbatch --parsable -p long"
+    batch_cmd = "sbatch --parsable -p long --mem 128G --cpus-per-task 16"
```

Comment:
We should use it otherwise it will run on one core, which takes a long time.
The training step works really well on multiple cores.
The best way would be to parse `n_jobs` from the lstchain config for sure

Reply:
If you set the cpu-per-task and pass `n_jobs=-1` in the lstchain config, will it not use all the cores you have, so precisely the number you set in the slurm config?

Reply:
There was no specific reason to set `n_jobs=4` in lstchain, it's just a reasonnable number but can be changed

Reply:
Yes that works as well, maybe even better :+1: 
Just if you set 16 for slurm and 4 in the config, 12 will be unused.
But really there is no way to not do -1 in the config

Reply:
> There was no specific reason to set `n_jobs=4` in lstchain, it's just a reasonnable number but can be changed

Its not even reasonable for our use case, because we only requested one core from slurm.
16 is the upper range of what still provides a speedup from my tests, below 8 is the largest difference